templates/KpisList/makedata.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { HvTooltip, HvTypography } from "@hitachivantara/uikit-react-core";
+import { Level0Good, Level1, Level2Average, Level3Bad } from "@hitachivantara/uikit-react-icons";
+
+const getEntry = (opts) => opts[Math.floor(Math.random() * opts.length)];
+
+function getDate() {
```

Comment:
could you transform this function to be in line with the others `const getDate = () => {}`